app/assets/stylesheets/application.scss

```diff
@@ -10,3 +10,4 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+@import "components/_card_for_superpowers";
```

Comment:
Could we name this file something more generic like just _card.scss, since we might re-use parts of it for other types of cards?